phpunit.xml

```diff
@@ -7,8 +7,7 @@
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
          processIsolation="false"
-         stopOnFailure="false"
-         syntaxCheck="false">
```

Comment:
why have we removed this syntax check?

Reply:
That argument has been removed from the last versions of PHPUnit, see the [answer by PHPUnit author here](https://stackoverflow.com/questions/44328114/phpunit-what-does-syntaxcheck-configuration-parameter-stands-for-exactly). I googled all I could and I can't even find what exactly that was doing 😕